docs/developer/releasing.md

```diff
@@ -40,8 +45,10 @@ For version v0.x.y:
    images and fat manifest with `REGISTRY="gcr.io/k8s-cluster-api" make all-push` (on the 0.1 release branch, we
```

Comment:
I think this needs adjusting?

Reply:
It applies to master rather than the `release-0.1` branch, but I went ahead and updated here as well.